frontend/src/App.js

```diff
@@ -33,73 +46,75 @@ const LoginContainer = () => (
   </div>
 )
 
-const ClientContainer = () => (
 
-  <div className="App-header" style={{ backgroundColor: '#CCCCCC' }}>
+const ClientContainer = () => (
+  <div className="App-header clientadvisorBackground">
     <div className="App-intro">
       <div>
         <Router>
           <Route render={({ location, history }) => (
             <div className="container">
               <SideNav
                 onSelect={(selected) => {
+
                   const to = '/' + selected;
                   if (location.pathname !== to) {
                     history.push(to);
                   }
-
                 }} >
                 <Center>
-                  <img src={Logo} rel="icon" alt="" style={{ width: '50px', paddingTop: '10px', paddingBottom: '5px' }} />
+                  <img src={Logo} rel="icon" alt="" style={logo} />
                 </Center>
                 <SideNav.Nav>
-                  <NavItem eventKey="clientform" >
-                    <NavIcon classnastyle={{ flexDirection: 'column' }} >
+                  <NavItem eventKey="clientform" active={location.pathname === '/clientform'} >
+                    <NavIcon>
                       <FontAwesomeIcon icon={faUser} />
-                      <span style={{ fontSize: '10px', position: 'absolute', top: '15px', left: '17px' }}>Profile</span>
+                      <span style={{ fontSize: '0.625rem', position: 'absolute', top: '0.9375rem', left: '1.0625rem' }}>Profile</span>
                     </NavIcon>
 
                   </NavItem>
 
-                  <NavItem eventKey="clientplan" >
+                  <NavItem eventKey="clientplan" active={location.pathname === '/clientplan'}>
                     <NavIcon>
                       <FontAwesomeIcon icon={faChartBar} />
-                      <span style={{ fontSize: '10px', position: 'absolute', top: '15px', left: '22px' }}>Plan</span>
+                      <span style={{ fontSize: '0.625rem', position: 'absolute', top: '0.9375rem', left: '1.375rem' }}>Plan</span>
```

Comment:
this should be a const css at the top of this file but nice job on the px -> rem